fastlane/Fastfile

```diff
@@ -0,0 +1,169 @@
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.146.0"
+
+skip_docs
+
+desc "Create a new FinniversKit version"
+lane :make_finniverkit_version do |options|
+  make_release("FinniversKit", "", options)
+end
+
+desc "Create a new FinnUI version"
+lane :make_finnui_version do |options|
+  make_release("FinnUI", "finnui-", options)
+end
+
+desc "Attempt to connect to github.com with ssh"
+lane :verify_ssh_to_github do
+  githuburl = "github.com"
+  UI.header "Verify ssh is setup and #{githuburl} can be accessed, will prompt for ssh key password if needed"
+  command = "ssh -T git@#{githuburl}"
+  sh(command, log: true, error_callback: ->(result) {
+    sshresult = result
+    if sshresult.include? "successfully authenticated"
+      UI.success "Could authenticate to #{githuburl}"
+    else
+      UI.shell_error! "Seems connection to #{githuburl} failed"
+    end
+    })
+  end
+
+  def make_release(project, tag_prefix, options)
```

Comment:
Just to clear things up, the methods below should be moved out of the lane `verify_ssh_to_github` and into the global scope instead

Reply:
Hehe, there was an indent error, fixed it, thanks 👍